src/planscape/planning/views.py

```diff
@@ -520,7 +602,6 @@ def create_scenario(request: HttpRequest) -> HttpResponse:
         if not result:
             return HttpResponse(
                 json.dumps({"reason": reason}),
-                content_type="application/json",
```

Comment:
we dont need this anymore?

Reply:
Oops,  this should actually converted to a Response() function, which returns data based on what is asked for, but I overlooked updating this one. Thanks for catching it.